### ISSUE TITLE
Alpha in neon2d.Colour not working correctly.

### ISSUE THREAD

**Issue by DontBelieveMe** [neon2d/neon2d#47]
I don't know why but currently the alpha channel seems to be stuck at 255 and its not being properly converted to hex in `Colour.getARGB()`.
I think it might be something to do with storing the alpha channel internally as a long - the only reason its a long is because it cannot be bit shifted as an int. I would use `uint` but that's generally bad in C#.
I have marked it as `[Broken]` in the code.

**Comment by matthewd673** [comment]
Could this be affecting #46, which failed it's AppVeyor build?

**Comment by DontBelieveMe** [comment]
No, The AppVeyor fail was related to trying to find `TemporaryFiles` that were not pushed by the gitignore.
